components/register-cluster/register_cluster.sh

```
@@ -0,0 +1,31 @@
+export BTOKEN=$(kubectl get secret cluster-argocd-manager-token -n kube-system -o jsonpath="{.data.token}" | base64 --decode)
+
+# Set namespace variables
+TPA_NAMESPACE="tssc-tpa"
+REALM="chicken"
+
+# Derive annotation values
+export APP_DOMAIN_URL="-${TPA_NAMESPACE}.${INGRESS_DOMAIN}"
+export KEYCLOAK_HOST="sso.${INGRESS_DOMAIN}"
+export OIDC_ISSUER_URL="https://sso.${INGRESS_DOMAIN}/realms/${REALM}"
+export REDIRECT_URI1=https://server${APP_DOMAIN_URL}
+export REDIRECT_URI2=https://server${APP_DOMAIN_URL}/*
```

Shellcheck (0.11.0)

[warning] Brace expansions and globs are literal in assignments. Quote it or use an array.

(SC2125)